js/ug_onEachFeature_fns.js

```diff
@@ -171,10 +171,9 @@ function icu_OEF(layer) {
 }
 
 function water_points_OEF(layer) {
-  layer.bindPopup('<strong>Name:</strong> ' + layer.feature.properties.name + '<br>' + '<strong>District:</strong> ' +
-      layer.feature.properties.addr_district + '<br>' + '<strong>SubCounty:</strong> ' +
-      layer.feature.properties.addr_subcounty + '<br>' + '<strong>Parish:</strong> ' +
-      layer.feature.properties.addr_parish + '<br>' + '<strong>Water Source Coverage (%):</strong> ' +
+  layer.bindPopup('<strong>District:</strong> ' +
+      layer.feature.properties.districts1_DName2016 + '<br>' + '<strong>Total Population:</strong> ' +
```

Comment:
Hi @lynnnakiganda ,

These fields in the districts data do not match `Total Population` and `Water Source Coverage`

Reply:
dont match how

Reply:
`districts1_DName2016` is the name of a district but `Total Population` should be a value if I'm not mistaken...

Reply:
TotalPopn thats total population and water1_water_percentage is water coverage

Reply:
That is not what is written in that section...